examples/speaker_recognition/hi-mia_eval.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import brentq
+from sklearn.metrics import roc_curve
+from tqdm import tqdm
+
+
+def get_acc(data_root='./myExps/hi-mia/', emb='', emb_labels='', emb_size=512):
+    basename = os.path.dirname(emb)
+    X_test = np.load(emb)
+    label_files = np.load(emb_labels)
+
+    assert len(X_test) == len(label_files)
+    trail_file = root + 'trials_1m'
+
+    test_list = {}
+    speaker_list = {}
+
+    for idx, line in enumerate(label_files):
+        line = line.strip()
+        speaker = line.split('.')[0].split('_')[0]
+        test_list[line] = idx
+
+        if speaker in speaker_list:
+            speaker_list[speaker].append(idx)
+        else:
+            speaker_list[speaker] = [idx]
+
+    emb = int(emb_size)
+    # import ipdb; ipdb.set_trace()
+    tmp_file = open(trail_file, 'r').readlines()
+    trail_score = open('trial_score.txt', 'w')
+
+    trial_embs = []
+    keys = []
+    all_scores = []
+    all_keys = []
+
+    for line in tqdm(tmp_file):
```

Comment:
Can you comment on what this loop is doing?

Reply:
Done

Reply:
done